Metadata/SFBAudioMetadata.m

```diff
@@ -601,7 +601,7 @@ - (void)copyAttachedPicturesFrom:(SFBAudioMetadata *)metadata
 - (NSArray *)attachedPicturesOfType:(SFBAttachedPictureType)type
 {
 	NSMutableArray *pictures = [NSMutableArray array];
-	for(SFBAttachedPicture *picture in _pictures) {
+    for(SFBAttachedPicture *picture in _pictures.copy) {
```

Comment:
I'm unclear on the reasoning behind this change. At least in the context of this PR I don't think it belongs?

Reply:
sorry, i submit the wrong place. It must be in the removeAttachedPicturesOfType mehtod